android/demo-app/src/main/java/com/stadiamaps/ferrostar/AppModule.kt

```diff
@@ -87,7 +87,7 @@ object AppModule {
                 NavigationControllerConfig(
                     StepAdvanceMode.RelativeLineStringDistance(
                         minimumHorizontalAccuracy = 25U, automaticAdvanceDistance = 10U),
-                    RouteDeviationTracking.StaticThreshold(15U, 25.0),
+                    RouteDeviationTracking.StaticThreshold(15U, 50.0),
```

Comment:
Yeah, I keep thinking about that... I don't know what a good default is yet since it probably varies somewhat based on what filtering and other things you're doing (if any), but we can probably offer some defaults like this eventually.